papa/src/main/java/papa/internal/Perfs.kt

```diff
@@ -198,6 +199,13 @@ internal object Perfs {
           initCalledCurrentTimeMillis - lastTime
         }
       }
+    val lastAppAliveElapsedTimeMillis = if (Build.VERSION.SDK_INT >= 30) {
```

Comment:
@pyricau wdyt? this accounts for the fact that getHistoricalProcessExitReasons will return an empty list on first run, then will always have a non-zero value for timestamp on subsequent runs.